lib/presentation/widgets/assistant_widget.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/generated/assets/assets.gen.dart';
+import 'package:water_tracker/presentation/widgets/calendar_widget.dart';
+import 'package:water_tracker/presentation/widgets/custom_icon_button_widget.dart';
+
+class AssistantWidget extends StatelessWidget {
+  const AssistantWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        CustomIconButtonWidget(onTap: () {}, image: Image.asset(Assets.images.customCloseIcon.path)),
+        const Spacer(),
```

Comment:
i think. we already implemented row like this without spacers. They are not perfectly laying for this type of UI. Use appbar instead?

Reply:
You are sure? It's just that if you use the App bar, the size will not be accurate, and it will be noticeable in the distance between transitions, I think so